src/components/interlocutor/screens/InterlocutorList/containers/InterlocutorListContainer.js

```diff
@@ -4,24 +4,41 @@ import { getInterlocutorList } from '../actions';
 import InterlocutorListView from '../views/InterlocutorListView';
 
 class InterlocutorListContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { searchActive: 0 };
+    this.updateInterlocutorListView = this.updateInterlocutorListView.bind(this);
+  }
+
   componentDidMount() {
     this.props.getInterlocutorList();
   }
 
+  updateInterlocutorListView() {
```

Comment:
Name of the method is not explicit
Plus, is it normal that you never can disable searchActive ?

Reply:
I changed the name of the method to : updateInterlocutorListViewFromSearch
The name searchActive is not correct : it just informs the InterlocutorListContainer which interlocutorList to take : the whole list from the props interlocutorListAll (came from the state interlocutor) or the list from the props interlocutorListFromRequest(came from the state search)

but it doesn't disable the bar search